autoscoper/src/ui/AutoscoperMainWindow.cpp

```diff
@@ -952,13 +953,28 @@ void AutoscoperMainWindow::save_tracking_results(QString filename)
     bool interpolate = diag->diag->radioButton_InterpolationSpline->isChecked();
     int volume = -1;
     if (diag->diag->radioButton_VolumeCurrent->isChecked()) volume = tracker->trial()->current_volume;
+    exportLogFile();
```

Comment:
The function name `exportLogFile` is too generic. I suggest to use  `logTrackingParameters`